wiki/Game_Modifiers/en.md

```diff
@@ -228,25 +228,25 @@ Using this mod on osu!mania will set the play as unranked play.
 
 > Miss a note and fail.
 
-Sudden Death mod causes the player to fail the map if the [combo](/wiki/Glossary/#combo) is ever broken, creating an extra large miss figure (only in osu!standard mode only) and a full HP drain.
+Sudden Death mod causes the player to fail the map if the [combo](/wiki/Glossary/#combo) is ever broken, resulting in an extra large miss figure (osu!standard mode only) and a full HP drain.
 
 Combo can be broken by missing a note, not clicking on a starting slider circle (osu!standard only), not collecting a slider tick, or failing to complete a spinner. However, combo will not be broken by missing a slider end (result in getting 100 score value instead for that slider completion; osu!standard only) or not holding/collecting on the empty slider path.
 
 #### Perfect
 
 ![Perfect (PF) mod icon][PF]
 
-Click on Sudden Death again to get Perfect mod.
+Click on Sudden Death again to activate Perfect mod.
 
 > Score Multiplier: 1.00x
 
 > Shortcut key: `SS` (from unselected) or `Shift`+`S`
 
 > SS or quit.
 
-Perfect mod causes _any_ imperfect hit (as in, any score values that does not give 100% accuracy) to be counted as a **miss** and the song will **forcefully restart automatically**; something Sudden Death does not do.
+Perfect mod causes _any_ imperfect hit (as in, any score values that do not give 100% accuracy) to be counted as a **miss** and the song will **forcefully restart automatically**; something Sudden Death does not do.
```

Comment:
* 'any score value that does not..." to agree with 'hit' prior to the parenthetical phrase
* 'and forcefully restarts the song automatically, something..."
** don't need the semicolon and this fits the verb tenses